archaius2-core/src/main/java/com/netflix/archaius/config/PropertiesConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.netflix.archaius.config;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertiesConfig extends MapConfig {
+
+    public static PropertiesConfig fromFile(String filename) throws IOException {
```

Comment:
@brharrington Thanks for the input.  I could modify the implementation to consider classpath as well as file system.  Would that work?  

What, in your opinion, should be the expected behavior if there are multiple matching files?  Should it fail with a multiple files exception, should it load all, should it use the first?